pkgs/development/libraries/rabbitmq-java-client/default.nix

```diff
@@ -1,52 +1,67 @@
 {
-  fetchurl,
   lib,
-  stdenv,
-  ant,
-  jdk,
-  jre,
-  python2,
-  makeWrapper,
+  maven,
+  fetchFromGitHub,
+  linkFarm,
+  python3,
+  ...
```

Comment:
You should be able to remove the entry from `pkgs/top-level/all-packages.nix` (https://github.com/NixOS/nixpkgs/blob/6fe4a66c78e3eb7d6fe13fe8d205ea4ff17dce55/pkgs/top-level/all-packages.nix#L10241-L10244) and move this to `pkgs/by-name/ra/rabbitmq-java-client`, making this ellipsis unnecessary.
```suggestion

```